### PR TITLE
OSDOCS-8327: Documented the OCI platform RN

### PR DESCRIPTION
**Pull request by dfitzmau**
[OSDOCS-8327](https://issues.redhat.com/browse/OSDOCS-8327)

Version(s):
4.14

Link to docs preview:
[Oracle® Cloud Infrastructure](https://66774--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-14-release-notes#ocp-4-14-oci)

QE review:
- [x] QE not needed for Dev Preview feature. See comment on https://issues.redhat.com/browse/OSDOCS-7595

Additional information:

### CODE REVIEW

File: release_notes/ocp-4-14-release-notes.adoc [hunk 1]
```diff
@@ -550,6 +550,18 @@ For information about using the {secrets-store-operator} to mount secrets from a
 
 For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-azure-file.adoc#persistent-storage-csi-azure-file-nfs_persistent-storage-csi-azure-file[NFS support].
 
+[id="ocp-4-14-oci"]
+=== Oracle&#174; Cloud Infrastructure
+
+You can now install an {product-title} cluster on {oci} by using either the Assisted installer or the Agent-based installer. For {product-title} {product-version}, {product-title} on {oci} is available as a Developer Preview feature.
```
Comment (tmalove) [inline release_notes/ocp-4-14-release-notes.adoc:556]:
```suggestion
You can now install an {product-title} cluster on {oci} by using the Assisted installer or the Agent-based installer. For {product-title} {product-version}, {product-title} on {oci} is available as a Developer Preview feature.
```

**Review by tmalove — APPROVED** [review]

File: release_notes/ocp-4-14-release-notes.adoc [hunk 2]
```diff
@@ -550,6 +550,18 @@ For information about using the {secrets-store-operator} to mount secrets from a
...
+
+To install an {product-title} cluster on {oci}, choose one of the following installation options:
+
+* link:https://access.redhat.com/articles/7039183[Using the Assisted Installer to install a cluster on {oci-first}]
```
Comment (tmalove) [inline release_notes/ocp-4-14-release-notes.adoc:560]: These  2 links do not resolve. Are they supposed to resolve now or after GA (release notes)?

**Review by tmalove — APPROVED** [review]

File: release_notes/ocp-4-14-release-notes.adoc (hunk 2 above)
Comment (dfitzmau) [reply release_notes/ocp-4-14-release-notes.adoc:560]: HI, @tmalove . Correct. The article are not published just yet. They must wait until GA.

**Review by pawanpinjarkar — APPROVED** [review]
/approve

File: release_notes/ocp-4-14-release-notes.adoc (hunk 2 above)
Comment (jeana-redhat) [reply release_notes/ocp-4-14-release-notes.adoc:933]: I can see the drafts, they are the correct links :+1:

File: release_notes/ocp-4-14-release-notes.adoc [hunk 3]
```diff
@@ -923,6 +923,18 @@ For information about using the {secrets-store-operator} to mount secrets from a
 
 For more information, see xref:../storage/container_storage_interface/persistent-storage-csi-azure-file.adoc#persistent-storage-csi-azure-file-nfs_persistent-storage-csi-azure-file[NFS support].
 
+[id="ocp-4-14-oci"]
+=== Oracle&#174; Cloud Infrastructure
+
+You can now install an {product-title} cluster on {oci} by using the Assisted installer or the Agent-based installer. For {product-title} {product-version}, {product-title} on {oci} is available as a Developer Preview feature.
```
Comment (jeana-redhat) [inline release_notes/ocp-4-14-release-notes.adoc:929]: Should the full name be used in this first instance?
```suggestion
You can now install an {product-title} cluster on {oci-first} by using the Assisted installer or the Agent-based installer. For {product-title} {product-version}, {product-title} on {oci} is available as a Developer Preview feature.
```

**Review by jeana-redhat — COMMENTED** [review]
Just one small thing here, feel free to ping me directly instead of re-queuing when ready

/remove-label merge-review-in-progress
/remove-label merge-review-needed

File: release_notes/ocp-4-14-release-notes.adoc (hunk 3 above)
Comment (dfitzmau) [reply release_notes/ocp-4-14-release-notes.adoc]: Good point, Updated.

**Pull request merged**